apps/ideaspace/src/components/modules/BrowseIdeas/BrowseIdeas.js

```diff
@@ -57,33 +62,50 @@ function BrowseIdeas() {
   }, []);
 
   return (
-    <PageWrapper>
-      {loading === true ? (
-        <CircularIndeterminateLoader text="Loading..." color="black" />
-      ) : (
-        <div>
-          <FilterDiv>
-            <label htmlFor="sortBy">Sort By</label>
-            <SortableDropdown
+    <>
+      <HeadWapper>
+        <Headline>Browse Ideas</Headline>
+        <StyledRanbow>
+          <RainbowBar width="100%" height="5px" />
```

Comment:
can we avoid using this explicit in-line styling?

Reply:
sure. they update the share component, something fix the need now.